modules/virt-cnv-network-alerts.adoc

```diff
@@ -48,3 +92,116 @@ $ oc logs -n $KMP_NAMESPACE $KMP_NAME
 .Resolution
 
 Open a support issue and provide the information gathered in the troubleshooting process.
+
+//KubeMacPoolDuplicateMacsFound Alert
+[id="KubeMacPoolDuplicateMacsFound_{context}"]
+== KubeMacPoolDuplicateMacsFound alert
+
+.Description
+
+The KubeMacPool allocates MAC Addresses and prevents MAC Address conflicts. After opening the KubeMacPool, a scan of the cluster for MAC Addresses in virtual machines set on the managed namespaces occurs. This alert occurs if duplicate MAC addresses are found in the scan.
```

Comment:
Thanks for the PR
I wonder wouldn't it be better to have it in one place and then refer it ?

For example this one, we can add missing info there if any and refer it,
another option is to create a tool that merge all those runbooks into this adoc (conversion would be needed).

https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/KubeMacPoolDuplicateMacsFound.md

Reply:
There is a plan to have to add automation for this process of u/s to d/s and d/s to u/s runbooks updates.

Reply:
Ok thanks, meanwhile can we strive please to unify the text between this doc and the run book ?
one example is the Description is bit different from here and the runbook 
```
KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.
Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.
In case it finds duplicates it will raise this alert.
```